packages/ethereum-wallets/src/lib/index.ts

```
@@ -860,6 +870,30 @@
           logger.log("Wallet already connected");
         }
 
+        console.log(
```

ESLint

[error] Unexpected console statement.

(no-console)